### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by RodwellSmith** [johngodley/redirection#666]
```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 5.3.6
Browser: Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36

Error: Unexpected token < in JSON at position 0
Action: redirection/v1/setting GET
Code: 404 Not Found
Raw: <h1 style='color:#497A97;font-size:12pt;font-weight:bold'>404 - Not Found
```

**Comment by johngodley** [comment]
Can you try the 3.1 beta? It should help with this problem:

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists
